reference/TubePlayer/Directory.Packages.props

```diff
@@ -29,7 +29,7 @@
     <PackageVersion Include="Uno.Wasm.Bootstrap" Version="8.0.4" />
     <PackageVersion Include="Uno.Wasm.Bootstrap.DevServer" Version="8.0.4" />
     <PackageVersion Include="Uno.Wasm.Bootstrap.Server" Version="7.0.31" />
-    <PackageVersion Include="Uno.WinUI" Version="5.0.64" />
+    <PackageVersion Include="Uno.WinUI" Version="5.1.0-dev.1268" />
```

Comment:
@nickrandolph, @jeromelaban 
I think the other Uno.WinUI versions should match the same version as well (Uno.WinUI....). 
Also, the fix (https://github.com/unoplatform/uno/pull/14902) should be backported no ? 
So we can use a proper stable version for the TubePlayer sample app but also the same version update will be needed for the SimpleCalc sample app (cf. screenshot and details here: https://github.com/unoplatform/uno.toolkit.ui-private/issues/7)

Reply:
We shouldn't be using dev/prerelease versions. We should either wait for 5.1 stable or backport.

Reply:
Indeed, let's wait for the 5.1 release. Let's not backport either for now, it's a very significant change.